lib/perl/Genome/Model/ReferenceAlignment.pm

```diff
@@ -459,7 +459,12 @@ sub get_lane_qc_models {
         my $lane_qc_model_name = $self->default_lane_qc_model_name_for_instrument_data($instrument_data);
 
         my $existing_model = Genome::Model->get(name => $lane_qc_model_name);
-        if ($existing_model) {
+        my $has_current_qc =
+            $existing_model &&
+            $existing_model->analysis_project_bridges &&
+            $existing_model->analysis_project_bridges->config_profile_item &&
+            $existing_model->analysis_project_bridges->config_profile_item->is_current;
+        if ($has_current_qc) {
             $self->debug_message("Default lane QC model " . $existing_model->__display_name__ . " already exists.");
```

Comment:
This should definitely be simplified!  For the moment it mirrors [a similar check in GenotypeMicroarray](https://github.com/genome/genome/blob/a4f871bd95af2f81dc6ccd509a26826e4bb29e38/lib/perl/Genome/Model/GenotypeMicroarray.pm#L177-L181).

I've got a branch that depends on this PR and #894 that will add a new status to models and will reduce the cascade of arrows here!